web_src/js/features/comp/Cropper.ts

```diff
@@ -0,0 +1,61 @@
+import Cropper from 'cropperjs';
```

Comment:
You need to lazy-load this dependency with `await import()`, just grep the codebase on how it's done.